controller/userProfileController.js

```diff
@@ -0,0 +1,36 @@
+let updateUser = require('../model/updateUserProfile.js')
+let getUser = require('../model/getUserProfile.js')
+
+const updateUserProfile = async (req, res) => {
+  try {
+    const { username, first_name, last_name, email, contact_number } = req.body;
+    if (!username) {
+      return res.status(400).send('Username is required');
+    }
+
+    const user_profile = await updateUser(username, first_name, last_name, email, contact_number)
```

Comment:
This is too many variables to pass down to a function. You already have all of these properties in the req.body you could just pass it down as an object. (or ideally make a copy of the body and pass that down instead).

Reply:
resolved